MobilityData/mobility-feed-api
# feat: removed `order_by` param filtering

**Summary:**
Closes #286

**Expected behavior:** 
Filtering using the `order_by` parameter is no longer supported.

**Testing tips:**
- Checkout branch
- Run api
- Send requests to `v1/gtfs_feeds` -- you should no longer have the option to filter using `order_by`

Please make sure these boxes are checked before submitting your pull request - thanks!
